environments/common/inventory/group_vars/builder/defaults.yml

```diff
@@ -3,4 +3,5 @@
 # as dependent on alphabetical ordering of groups, so if these variables are
 # defined elsewhere the group that is ordered lower will determine the values.
 openhpc_slurm_service_started: false
-nfs_client_mnt_state: present
\ No newline at end of file
+nfs_client_mnt_state: present
+openhpc_rebuild_reconfigure: false
```

Comment:
I wonder whether we should not this:

https://github.com/stackhpc/ansible_collection_slurm_openstack_tools/blob/main/roles/rebuild/tasks/validate.yml#L12

as well. Just thinking if we are building the image in CI then the runner might not have access to openstack.

Reply:
Good point. That role needs altering anyway though to move openstack creds off the compute node (https://github.com/stackhpc/ansible_collection_slurm_openstack_tools/issues/27) so will leave it for now.